05-Load-Balancer/main.tf

```diff
@@ -26,7 +26,7 @@ provider "google" {
  * - version: "~> 3.1"
  * - project: var.project_id
  * - name: "lab05-http-load-balancer"
- * - firewall_networks: module.network.network_self_link
+ * - firewall_networks: [module.network.network_self_link]
```

Comment:
I feel like this is something that a user should  figure out by looking at the [documentation](https://github.com/terraform-google-modules/terraform-google-lb-http) that the input should be a list.